app/controllers/kondos_controller.rb

```diff
@@ -7,12 +7,21 @@ def index
     if !user_signed_in? || current_user.renter?
       @kondos = policy_scope(Kondo).order(created_at: :desc)
 
+      #Homepage search
       unless params[:search_kondos] == ""
         @kondos = policy_scope(Kondo).where("LOWER(prefecture) like ?","%#{params[:search_kondos].to_s.downcase}%")
       else
         @kondos = policy_scope(Kondo).all
       end
 
+      #Index tab search
+      
+      if params[:search]
+         @filter = params[:search]["tags"].push(params[:search]["prefecture"]).flatten
+         @kondos = Kondo.all.global_search("#{@filter}")
```

Comment:
```suggestion
         @kondos = policy_scope(Kondo).all.global_search("#{@filter}")
```
I know your love for pundit :P